std_msgs/Header header

float32 longitudinal
float32 lateral
float32 vertical
float32 yaw
float32 pitch
uint8 plugOrientation
float32 length
float32 width
float32 height
float32 wheelbase
uint16 maximumDistance
uint8 frequencySlot
uint8 cycleTime
uint8 timeSlot
uint8 hcc
uint8 powerSaveStandstill
uint32 sensorIPAddress0
uint32 sensorIPAddress1
uint8 configurationCounter
uint8 statusLongitudinalVelocity
uint8 statusLongitudinalAcceleration
uint8 statusLateralAcceleration
uint8 statusYawRate
uint8 statusSteeringAngle
uint8 statusDrivingDirection
uint8 statusCharacteristicSpeed
uint8 statusRadar